src/gcp/cloudfunctions.js

```diff
@@ -65,13 +65,19 @@ function _createFunction(options) {
   const location = "projects/" + options.projectId + "/locations/" + options.region;
   const func = location + "/functions/" + options.functionName;
   const endpoint = "/" + API_VERSION + "/" + location + "/functions";
+
   const data = {
     sourceUploadUrl: options.sourceUploadUrl,
     name: func,
     entryPoint: options.entryPoint,
     labels: options.labels,
     runtime: options.runtime,
   };
+
+  if (options.vpcConnector) {
+    data.vpcConnector = options.vpcConnector;
+    data.vpcConnectorEgressSettings = options.vpcConnectorEgressSettings || "ALL_TRAFFIC";
```

Comment:
As with the change in `firebase-functions`, we should allow the server-side default here.

```suggestion
    if (options.vpcConnectorEgressSettings) {
      data.vpcConnectorEgressSettings = options.vpcConnectorEgressSettings;
    }
```

Reply:
Good catch, forgot this one. It's fixed.